core/src/main/scala/com/ing/baker/java_api/JBaker.scala

```diff
@@ -91,10 +91,8 @@ class JBaker private (jRecipe: JRecipe,
     *
     * @param processId the identifier of the process
     * @param event The event to fire
-    * @throws BakerException
     * @return
     */
-  @throws(classOf[BakerException])
```

Comment:
Correct me if I am wrong?  

We might get the following exceptions, right?

- CancellationException
- ExecutionException
-  InterruptedException
- TimeoutException